Alexa.NET/Alexa.NET.csproj

```diff
@@ -5,22 +5,22 @@
     <AssemblyTitle>Alexa.NET</AssemblyTitle>
     <VersionPrefix>1.1.0</VersionPrefix>
     <Authors>Tim Heuer</Authors>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netstandard1.4</TargetFramework>
     <AssemblyName>Alexa.NET</AssemblyName>
     <PackageId>Alexa.NET</PackageId>
     <PackageTags>amazon;alexa;echo;dot;echo dot;skills</PackageTags>
     <PackageReleaseNotes>Adding video app interface support.</PackageReleaseNotes>
     <PackageProjectUrl>https://github.com/timheuer/alexa-skills-dotnet</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/timheuer/alexa-skills-dotnet/blob/master/LICENSE</PackageLicenseUrl>
     <RepositoryUrl>https://github.com/timheuer/alexa-skills-dotnet</RepositoryUrl>
-    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
```

Comment:
I'd like to use this with AWS Lambda, and 1.6.0 is the latest version it supports. We can use Json.NET 10.0.1 instead.

Reply:
This branch has been tested with AWS Lambda and works correctly, as does all versions of Json.NET (as it still targets .NET Standard 1.3 I believe). All my personal Skills use Lambda, but I did just create a new Lambda package with the most recent AWS tooling and this branch of code to ensure that was still true.

The version bump you've highlighted is regarding the build of the package rather than its usage and is for the NETStandard.Library reference, which as a meta-package can have a higher version than the .NET Standard you're targeting and doesn't affect the build.

The best article I've seen explaining this is https://andrewlock.net/what-is-the-netstandard-library-metapackage/

If you look at the Appveyor build for this pull request, you'll be able to see the build successfully generates a package for .NET Standard 1.4 out of the code change I've made. I merged in the latest changes to ensure that this was still the case. AWS Lambda is compatible with 1.6 or below so this package should continue to work.

Hope this clears things up. If there's a specific issue or combination where this isn't the case - please send a link to the code or screenshot and I'll take a look.

Thanks for checking this with me.

Reply:
I'm getting the following error:

```
Executing publish command
... invoking 'dotnet publish', working folder 'C:\Users\xxx\Desktop\alexa-skills-dotnet\AWSLambda1\bin\Release\netcoreapp1.0\publish'
... publish: Microsoft (R) Build Engine version 15.4.8.50001 for .NET Core
... publish: Copyright (C) Microsoft Corporation. All rights reserved.
... publish:   Alexa.NET -> C:\Users\agodong\Desktop\alexa-skills-dotnet\Alexa.NET\bin\Release\netstandard1.4\Alexa.NET.dll
... publish:   AWSLambda1 -> C:\Users\agodong\Desktop\alexa-skills-dotnet\AWSLambda1\bin\Release\netcoreapp1.0\AWSLambda1.dll
... publish:   AWSLambda1 -> C:\Users\agodong\Desktop\alexa-skills-dotnet\AWSLambda1\bin\Release\netcoreapp1.0\publish\
Error: Project is referencing NETStandard.Library version 1.6.1. Max version supported by netcoreapp1.0 is 1.6.0.
Error: NETStandard.Library 1.6.1 is used for target framework netcoreapp1.1.
Error: Check the following dependencies for versions compatible with netcoreapp1.0:
Error: 	Amazon.Lambda.Serialization.Json : 1.1.0
Error: 	Alexa.NET : 1.2.0
```

Repro steps:
1. Check out this branch
2. Create a new AWS Lambda project
3. Publish project

.csproj for (2):
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Amazon.Lambda.Core" Version="1.0.0" />
    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="1.1.0" />
  </ItemGroup>

  <ItemGroup>
    <DotNetCliToolReference Include="Amazon.Lambda.Tools" Version="1.8.0" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\Alexa.NET\Alexa.NET.csproj" />
  </ItemGroup>

</Project>
```

Reply:
Ahh okay - a local reference would certainly be different. Can you confirm it works if you use the .NET Standard 1.4 output from the debug/release folder just to make sure my machine isn't totally broken :smile: 

I'll bring it back down so it works for local development as well - thanks for the extra detail @adriangodong

Reply:
Not just local reference, if I package-reference Json.NET 10.0.3 to the Lambda csproj, it returns the same error on publish:

```
Error: Project is referencing NETStandard.Library version 1.6.1. Max version supported by netcoreapp1.0 is 1.6.0.
Error: NETStandard.Library 1.6.1 is used for target framework netcoreapp1.1.
Error: Check the following dependencies for versions compatible with netcoreapp1.0:
Error: 	Amazon.Lambda.APIGatewayEvents : 1.1.0
Error: 	Amazon.Lambda.Serialization.Json : 1.1.0
Error: 	Newtonsoft.Json : 10.0.3
Error: 	GitHubHook : 0.0.0
```

If package-reference it to a library pinned to 1.6.0, Nuget returns package downgrade error:
```
Detected package downgrade: NETStandard.Library from 1.6.1 to 1.6.0. Reference the package directly from the project to select a different version. 
 GitHubHook -> Newtonsoft.Json 10.0.3 -> NETStandard.Library (>= 1.6.1) 
 GitHubHook -> NETStandard.Library (>= 1.6.0)	GitHubHook	C:\Users\xxx\Desktop\GitHubHook\GitHubHook\GitHubHook.csproj	1	
```

I'm not crazy! 😄

Reply:
I'm also having problem with a different package and I've requested the author to pin it to 1.6.0. This comment has some of my observation and possible reason why the errors are happening: https://github.com/egorikas/UnixTimeConverter/pull/11#issuecomment-342394970